testing/test_config.py

```diff
@@ -352,6 +352,25 @@ def test_inifilename(self, tmpdir):
         assert config.inicfg.get('name') == 'value'
         assert config.inicfg.get('should_not_be_set') is None
 
+    @pytest.mark.issue1618
+    def test_consider_plugin(self, testdir):
+        pytest.importorskip('xdist')
```

Comment:
Are you sure? I'm think I tested this fix on a virtual environment without xdist...

Oh yes, I remember, I got the same problem and tested your fix just by running `py.test`, without passing `-n1`. If it should be run in both environments, I guess we should parametrize this then.